esmvalcore/cmor/_fixes/fix.py

```diff
@@ -57,24 +59,24 @@ def fix_file(
 
         Parameters
         ----------
-        filepath: Path
+        filepath:
             File to fix.
-        output_dir: Path
+        output_dir:
             Output directory for fixed files.
-        add_unique_suffix: bool, optional (default: False)
+        add_unique_suffix: optional
```

Comment:
why are you removing the types from the docstring? It's still nice to have them in the API docs, I don't think they get parsed from the functional definition, do they?

Reply:
Very good point. I added `autodoc_typehints = 'both'` to our RTD config - now they are properly shown in the function description, see https://esmvaltool--2160.org.readthedocs.build/projects/ESMValCore/en/2160/api/esmvalcore.preprocessor.html#esmvalcore.preprocessor.fix_file.

Before that, they have only been shown in the function signature at the very beginning.

Reply:
Danke, Manu!